src/components/panel/panel.tsx

```diff
@@ -0,0 +1,17 @@
+import { SidePanel } from "./side-panel";
+import {
+    SidePanelBody,
+    SidePanelContent,
+} from "./side-panel/side-panel.styles";
+
+export const Panel = () => {
+    return (
+        <SidePanel>
+            <SidePanelBody>
+                <SidePanelContent>
+                    <h2>Hello World!</h2>
+                </SidePanelContent>
+            </SidePanelBody>
```

Comment:
- SidePanelBody should be within the SidePanel component
- move `overflow-y` style from SidePanelContent to SidePanelBody
- understand SidePanelContent is likely just a placeholder component, I don't see much use of it at this point and think we can remove it altogether, just render `<h2>Hello World!</h2>` will do

Reply:
We've refactored the layout to make things clearer